CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 5.0.0
+
+### Breaking Changes
+
+- [#71](https://github.com/okta/okta-react/pull/71) Removes adding default `restoreOriginalUri` callback in `Security`
```

Comment:
nit: adding -> ~adding~

Reply:
The default `restoreOriginalUri ` from `auth-js` changes `window.location`, which reload the whole SPA, we should also accept an `restoreOriginalUri ` (required) callback from `<Security>` prop to override the default one from auth-js.